src/components/NoticeBox.js

```diff
@@ -1,17 +1,64 @@
 import React from "react";
 
 export function NoticeBox(props) {
-  if (!props.notices) {
+  let notices = [];
+
+  // check data is valid
+  if (props.hasValidData == false) {
+    notices.push([
+      "We weren’t able to properly locate your content for Search",
+      "noticeBoxRed",
```

Comment:
Allowing the classname to be defined is really flexible, but it looks like we only have two types of notice box:

* Normal notice
* Really important scary red notice

... so we could perhaps simplify by using a boolean instead (unless we think we'll have lots of different notice box types in the future).

How about `isImportant: true` or `isScary: true`?

p.s. It's not a joke - `isScary` is used in Calypso: https://github.com/Automattic/wp-calypso/search?q=isScary :)

Reply:
TIL about `isScary` 😆 and good idea, simplifying with a boolean makes a lot of sense here.